### PR TITLE
EWPP-3344: Fix restored revision.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: modules/oe_editorial_entity_version/src/Form/NodeRevisionRevertForm.php [hunk 1]
```diff
@@ -157,6 +157,12 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
     $this->revision->setChangedTime($this->time->getRequestTime());
     $this->revision->set('moderation_state', 'draft');
 
+    // With took over translation, the changed time should be
```
Comment (22Alexandra) [inline modules/oe_editorial_entity_version/src/Form/NodeRevisionRevertForm.php]: Carry over translations is not a functionality of this module so we shouldn't mention it. The comment could be updated to 
```suggestion
    // Set the same changed time for existing translations of the current revision.
```

**Review by 22Alexandra — DISMISSED** [review]

**Comment by 22Alexandra** [comment]
We need to lock symfony/validator package to avoid this https://drone.fpfis.eu/openeuropa/oe_editorial/702/30

File: modules/oe_editorial_entity_version/src/Form/NodeRevisionRevertForm.php (hunk 1 above)
Comment (sergepavle) [reply modules/oe_editorial_entity_version/src/Form/NodeRevisionRevertForm.php]: fixed

File: composer.json [hunk 2]
```diff
@@ -26,7 +26,8 @@
         "openeuropa/oe_paragraphs": "^1.19",
         "openeuropa/task-runner-drupal-project-symlink": "^1.0-beta6",
         "phpspec/prophecy-phpunit": "^2",
-        "symfony/phpunit-bridge": "^6.0"
+        "symfony/phpunit-bridge": "^6.0",
+        "symfony/validator": "~v4.4.48 || ~v6.2.5"
```
Comment (nagyad) [inline composer.json]: This change can be removed.

**Review by nagyad — APPROVED** [review]

**Review by nagyad — APPROVED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Pull request merged**